package/kernel/linux/modules/fs.mk

```diff
@@ -470,8 +472,7 @@ define KernelPackage/fs-nfs-common
   FILES:= \
 	$(LINUX_DIR)/fs/lockd/lockd.ko \
 	$(LINUX_DIR)/net/sunrpc/sunrpc.ko \
-	$(LINUX_DIR)/fs/nfs_common/grace.ko \
-	$(LINUX_DIR)/fs/nfs_common/nfs_ssc.ko
+	$(LINUX_DIR)/fs/nfs_common/grace.ko
```

Comment:
removing `nfs_ssc.ko` looks strange, the build should have failed when it is not there.

Reply:
I'll recheck it.

Reply:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/fs/Kconfig?h=linux-5.15.y#n360

NFS_V4_2_SSC_HELPER is now bool in 5.15+
It will never be built as nfs_ssc.ko. Instead, it will be treated as built-in, and will never fail the build.